### PR TITLE
[DeviceRegistry] Added Azure.ResourceManager.DeviceRegistry SDK

### PR DESCRIPTION
**Pull request by davidemontanari**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -964,3 +964,7 @@
 
 # Onboarding Documentation and Quickstarts
 /doc/DataPlaneCodeGeneration                                       @chunyu3 @pshao25 @lirenhe @annelo-msft
+
+# PRLabel: %Device Registry
```
Comment (jsquire) [inline .github/CODEOWNERS:968]: Please move this to the management library section as the first entry after the catch-all (L933) and follow the existing format.  (L933-937 is a good example)

Comment (jsquire) [reply .github/CODEOWNERS:968]: The account issues caught by the linter are confirmed to be fixed.

**Review by jsquire — DISMISSED** [review]

File: sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/src/Azure.ResourceManager.DeviceRegistry.csproj [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.ResourceManager.DeviceRegistry</PackageId>
+    <Description>Azure Resource Manager client SDK for Azure resource provider DeviceRegistry.</Description>
+    <PackageTags>azure;management;arm;resource manager;deviceregistry</PackageTags>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)ChangeTrackingList.cs" LinkBase="Shared/Core" />
```
Comment (davidemontanari) [inline sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/src/Azure.ResourceManager.DeviceRegistry.csproj:9]: For reference: had to add L9-10 to fix `dotnet build` errors in the generated code. The errors look like:

- `error CS0122: 'Optional<T>.Value' is inaccessible due to its protection level`
- `error CS0122  'ChangeTrackingList<T>' is inaccessible due to its protection level`

**Review by marcodalessandro — APPROVED** [review]

File: sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure DeviceRegistry management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/README.md:3]: Add service description here.

File: sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/CHANGELOG.md:3]: Specify the release date by replacing the `Unreleased` by the actual date, format is `yyyy-mm-dd`.

File: sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/src/Azure.ResourceManager.DeviceRegistry.csproj (hunk 2 above)
Comment (davidemontanari) [reply sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/src/Azure.ResourceManager.DeviceRegistry.csproj]: This has been fixed. Thanks @ArthurMa1978.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**